cmd/qat_plugin/README.md

```diff
@@ -1,39 +1,211 @@
+
+| Argument | Meaning |
+|:-------- |:------- |
+| -debug | enable debug output
+| -dpdk-driver string | DPDK Device driver for configuring the QAT device (default "vfio-pci") |
+| -kernel-vf-drivers string | Comma separated VF Device Driver of the QuickAssist Devices in the system. Devices supported: DH895xCC,C62x,C3xxx and D15xx (default "dh895xccvf,c6xxvf,c3xxxvf,d15xxvf") |
+| -max-num-devices int | maximum number of QAT devices to be provided to the QuickAssist device plugin (default 32) |
+| -mode string | plugin mode which can be either dpdk (default) or kernel (default "dpdk") |
+
+The example [DaemonSet YAML](../../deployments/qat_plugin/qat_plugin.yaml) passes a number of these
+arguments, and takes its default values from the
+[QAT default ConfigMap](../../deployments/qat_plugin/qat_plugin_default_configmap.yaml). The following
+table summarises the defaults:
+
+| Argument | Variable | Default setting | Explanation |
+|:-------- |:-------- |:--------------- |:----------- |
+| -dpdk-driver | `$DPDK_DRIVER` | vfio-pci | A more robust and secure choice than the `igb_uio` alternative |
+| -kernel-vf-drivers | `$KERNEL_VF_DRIVERS` | dh895xccvf,c6xxvf,c3xxxvf,d15xxvf | Modify to suit your hardware setup |
+| -max-num-devices | `$MAX_NUM_DEVICES` | 32 | Modify to suit your hardware setup if necessary |
+| -debug | `$DEBUG` | false | Debug is disabled by default |
+
+For more details on the `-dpdk-driver` choice, see 
+[DPDK Linux Driver Guide](http://dpdk.org/doc/guides/linux_gsg/linux_drivers.html).
+
+For more details on the available options to the `-kernel-vf-drivers` option, see the list of
+vf drivers available in the [Linux Kernel](https://github.com/torvalds/linux/tree/master/drivers/crypto/qat).
+
+> **Note**: `kerneldrv` mode is excluded from builds by default. See the
+> [Build the plugin image](#build-the-plugin-image) section for more details.
+
+If the `-mode` parameter is set to `kerneldrv`, no other parameter documented above are valid,
+except `-debug` which is global for both modes.
+`kerneldrv` mode implements resource allocation based on system configured [logical instances][7].
+
+The `kerneldrv` mode does not guarantee full device isolation between containers
+and therefore it's not recommended. This mode will be deprecated and removed once `libqat`
+implements non-UIO based device access.
+
+# Installation
+
+The below sections cover how to obtain, build and install this component.
+
+The component can be installed either using a DaemonSet or running 'by hand' on each node.
+
+## Pre-built image
+
+[Pre-built images](https://hub.docker.com/r/intel/intel-qat-plugin)
+of this component are available on the Docker hub. These images are automatically built and uploaded
+from the latest master branch of this repository.
+
+Release tagged images of the components are also available on the Docker hub, tagged with their
+release version numbers.
```

Comment:
should we also mention the release tagged images have the correspondent `release-X.Y` branch in this repo?